File: src/main/java/ua/softgroup/matrix/server/supervisor/consumer/json/ActiveProjectsJson.java
package ua.softgroup.matrix.server.supervisor.consumer.json;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class ActiveProjectsJson {

    @JsonProperty
    private boolean success;
    @JsonProperty
    private List<ProjectJson> list;

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public List<ProjectJson> getList() {
        return list;
    }

    public void setList(List<ProjectJson> list) {
        this.list = list;
    }

    @Override
    public String toString() {
        return "ActiveProjectsJson{" +
                "success=" + success +
                ", list=" + list +
                '}';
    }
}

File: src/main/scala/ua/softgroup/matrix/server/supervisor/consumer/json/ActiveProjectsJson.scala
package ua.softgroup.matrix.server.supervisor.consumer.json

import java.util

import com.fasterxml.jackson.annotation.JsonProperty

import scala.beans.{BeanProperty, BooleanBeanProperty}

/**
  * @author Oleksandr Tyshkovets <olexandr.tyshkovets@gmail.com>
  */
class ActiveProjectsJson {

  @JsonProperty
  @BooleanBeanProperty
  var success: Boolean = _

  @JsonProperty
  @BeanProperty
  var list: util.List[ProjectJson] = _

  override def toString = s"ActiveProjectsJson(success=$success, list=$list)"

}
